src/index.tsx

```diff
@@ -646,6 +646,17 @@ export function Root({
     }
   }
 
+  React.useEffect(() => {
+    if (!modal) {
+      // Need to do this manually unfortunately
+      if (typeof window !== 'undefined') {
```

Comment:
I don't think this check is needed since we are doing it in a `useEffect` now?

Reply:
True